docs/guides/breaking-changes.md

```
@@ -0,0 +1,365 @@
+
+```bash
+# Compare two schema files
+cvt compare --old ./v1/openapi.json --new ./v2/openapi.json
+
+# JSON output for scripting
+cvt compare --old ./v1/openapi.json --new ./v2/openapi.json --json
+```
+
+### Output Example
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)